web/src/containers/Sidebar.js

```diff
@@ -202,8 +202,8 @@ class Sidebar extends Component {
     return (
       <div className="ds-l-col--3">
         <aside className="site-sidebar">
-          <div className="xs-hide sm-hide site-sidebar__sticky">
-            <div className="flex items-center ds-u-border-bottom--1 ds-u-padding-y--2 ds-u-margin-bottom--4">
+          <div className="site-sidebar__sticky">
```

Comment:
Is it okay that the sidebar no longer disappears on narrow screens? My guess is this is a usability improvement because previously it went away entirely and there was no longer any way to quickly navigate. Maybe there should be a new issue to figure out how to handle the sidebar on small screens?

Question for @lauraponce maybe?

Reply:
Yeah, we should probably have issues to figure out how we want to deal with small screens. Right now, nothing really works well on a small screen (and I think there's an argument to be made that most of this doesn't need to work on a small screen).